app/webpacker/controllers/csv_header_report_controller.js

```diff
@@ -0,0 +1,19 @@
+import { Controller } from "stimulus";
+
+export default class extends Controller {
+  static targets = ["reportType"]
+
+  handleSelectChange() {
+    let displayHeaderRow = document.querySelector("#display_header_row");
```

Comment:
Using `querySelector` to get the element is not recommended with StimulusJs, why don't you use `targets` ?
https://stimulus.hotwired.dev/reference/targets